youtube_dl/extractor/atresplayer.py

```diff
@@ -132,9 +132,11 @@ def _real_extract(self, url):
                 'url': video_url,
                 'format_id': 'http',
             }
-            mobj = re.search(r'(?P<bitrate>\d+)K_(?P<width>\d+)x(?P<height>\d+)', video_url)
+            mobj = re.search(r'(?P<bitrate>\d+)K_(?P<width>\d+)x(?P<height>\d+)', video_url)\
+                    or re.search(r'video_(?P<width>\d+)(x(?P<height>\d+))?_(?P<bitrate>\d+)', video_url)
             if mobj:
                 format_info.update({
+                    'preference': -40,
```

Comment:
@dstftw not sure if that's appropriate (feel free to change). But IMO it didn't feel right that lower quality HTTP videos were display as best quality simply because the others were video only (when audio is always available).

[common.py (relevant bits)](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/common.py#L1112)
```
                if f.get('acodec') == 'none':  # video only
                    preference -= 40
```